utils/markdown/renderer.jsx

```diff
@@ -120,19 +119,40 @@ export default class Renderer extends marked.Renderer {
                 }
             }
         }
-        src = PostUtils.getImageSrc(src, this.formattingOptions.proxyImages);
-        let out = '<img src="' + src + '" alt="' + text + '"';
-        if (title) {
-            out += ' title="' + title + '"';
-        }
-        if (dimensions.length > 0) {
-            out += ' width="' + dimensions[0] + '"';
-        }
-        if (dimensions.length > 1) {
-            out += ' height="' + dimensions[1] + '"';
+
+        const regex = /.+\/(.+\.(gifv))(?:\?.*)?$/i;
+        const match = src.match(regex);
+        let out = '';
+
+        //If the URL src is a gifv, then render it in a <video> HTML5 element, otherwise it will go into an <img> element
+        if (match) {
+            const jpgSrc = src.substr(0, src.lastIndexOf('.gifv')) + '.jpg';
+            const mp4Src = src.substr(0, src.lastIndexOf('.gifv')) + '.mp4';
```

Comment:
Is it necessary to change the extension on the video source? I think `<video><source src="https://i.imgur.com/abc123.gifv"/></video>` might be valid

Reply:
I will look into this soon!